library/vulnerabilities/attack-wave-detection/isWebScanner.benchmark.test.ts

```diff
@@ -0,0 +1,39 @@
+import * as t from "tap";
+import { isWebScanner } from "./isWebScanner";
+import { type Context } from "../../agent/Context";
+
+function getTestContext(path: string, method: string, query = {}): Context {
+  return {
+    remoteAddress: "::1",
+    method: method,
+    url: `http://localhost:4000${path}`,
+    query: query,
+    headers: {
+      "content-type": "application/json",
+    },
+    body: {},
+    cookies: {},
+    routeParams: {},
+    source: "express",
+    route: path,
+  };
+}
+
+t.test("it does take less than 0.1ms", async (t) => {
+  const start = performance.now();
+
+  const iterations = 1000;
```

Comment:
```suggestion
  const iterations = 25_000;
```

I would increase this so that JIT kicks in?